steps/localazy/1.0.5/step.yml

```diff
@@ -0,0 +1,89 @@
+  updated translations.\n[Localazy](https://localazy.com) is a translation management
+  platform built by developers for developers \nto localize apps in the modern way. "
+website: https://localazy.com
+source_code_url: https://github.com/localazy/bitrise-step-localazy
+support_url: https://github.com/localazy/bitrise-step-localazy/issues
+published_at: 2021-08-27T10:46:27.924269874+02:00
+source:
+  git: https://github.com/localazy/bitrise-step-localazy.git
+  commit: 33819490a1282ad92b12408016d11b1bc40be874
+type_tags:
+- build
+deps:
+  brew:
+  - name: java
+  apt_get:
+  - name: openjdk-11-jre
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- config_file: localazy.json
+  opts:
+    description: |-
+      If you use different filename for the configuration file (not `localazy.json`), please specify the filename here.
+
+      For more information, visit [Localazy CLI - Command-Line Options](https://localazy.com/docs/cli/command-line-options).
+    is_required: false
+    summary: The filename of the configuration file. Default is `localazy.json`.
+    title: Configuration file
+- key_file: null
+  opts:
+    description: |-
+      If you provide read and write keys in the key file, please specify its filename here.
+
+      For more information, visit [Localazy CLI - Authorization](https://localazy.com/docs/cli/authorization).
+    is_required: false
+    summary: The filename of the key file
+    title: Key file
+- opts:
+    description: "If you don't provide read and write keys in the configuration or
+      key file, please provide it here.\n\nFor more information, visit [Localazy CLI
+      - Authorization](https://localazy.com/docs/cli/authorization) \nand [Localazy
+      CLI - Command-Line Options](https://localazy.com/docs/cli/command-line-options)."
+    is_expand: true
+    is_required: false
+    is_sensitive: true
+    summary: The custom read key
+    title: Read Key
+  read_key: null
+- opts:
+    description: "If you don't provide read and write keys in the configuration or
+      key file, please provide it here.\n\nFor more information, visit [Localazy CLI
+      - Authorization](https://localazy.com/docs/cli/authorization) \nand [Localazy
+      CLI - Command-Line Options](https://localazy.com/docs/cli/command-line-options)."
+    is_expand: true
+    is_required: false
+    is_sensitive: true
+    summary: The custom write key
+    title: Write Key
+  write_key: null
+- groups_list: null
+  opts:
+    description: |-
+      Groups allow for running only selected rules from the configuration file.
+
+      For more information, visit [Localazy CLI - Command-Line Options](https://localazy.com/docs/cli/command-line-options#groups).
+    is_required: false
+    summary: The list of download groups separated by space
+    title: Groups
+- opts:
+    description: 'You should invoke Localazy CLI in the directory containing the configuration
```

Comment:
Is this intentionally broken into two lines? I think you can remove the apostrophe and the extra whitespace at the end of the sentence.

Reply:
There are some extra whitespaces, but it's not broken on our side. Is it possible that your tool made some changes to formatting?